inventory-generation/tower-jobs-schedules/files/hosts

```diff
@@ -0,0 +1,2 @@
+[tower-management-host]
+localhost ansible_connection=local
```

Comment:
Same as before - I'd suggest removing the `ansible_connection` since it's defined in the `host_vars` area

Reply:
Same as before. This gets used by the next stage and is copied into the generated inventory with `copy`